src/questions/questions.js

```diff
@@ -221,7 +239,7 @@ export const QUICKSTART_ANSWERS = () => {
     quorumVersion: LATEST_QUORUM,
     transactionManager,
     cakeshop,
-    generateKeys: false,
+    tools: ['cakeshop'],
```

Comment:
this `generateKeys: false` line was deleted by accident. please remove the `cakeshop,` line above it instead

Reply:
good catch - fixed.